scripts/install.tpl.sh

```diff
@@ -38,5 +39,10 @@ kubectl_wait "cert-manager" "deployment/cert-manager-webhook" "60s"
 kubectl apply -f "https://github.com/operator-framework/catalogd/releases/download/${catalogd_version}/catalogd.yaml"
 kubectl_wait "olmv1-system" "deployment/catalogd-controller-manager" "60s"
 
+if [[ "${install_default_catalogs,,}" == "true" ]]
```

Comment:
```suggestion
if [[ "${install_default_catalogs,,}" == "true" ]]; then
```

Reply:
thanks, I threw this up right before lunch and didn't run it locally yet to catch that typo :)